include/evmc/filter_iterator.hpp

```diff
@@ -32,10 +33,19 @@ template <typename BaseIterator,
           bool predicate(typename std::iterator_traits<BaseIterator>::value_type) noexcept>
 struct filter_iterator
 {
+    /// The iterators' difference type.
```

Comment:
either `iterator's` or just `iterator`, make it consistent with comments below

Reply:
This was suppose to be "two iterators' difference type".

Reply:
Fine with changing to "Two"